docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -28,3 +28,65 @@ script will run on every unit. The script is assumed to be idempotent (in other
 can be run multiple times) and that it can be run on multiple units simultaneously
 without issue. Handling multiple migration scripts that run concurrently
 can be achieved by, for example, locking any tables during the migration.
+
+Use observability
+-----------------
+
+First, :ref:`integrate your web app with the Canonical Observability
+Stack <integrate_web_app_cos>`.
+
+Get Grafana login details
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+View the endpoints by running:
+
+.. code-block::
+
+    juju show-unit catalogue/0 | grep url
+    juju run grafana/leader get-admin-password
+
+These commands output URLs and the default admin password. Look for the URL
+with the Grafana postfix with the syntax
+``http://<IP_ADDRESS>/<JUJU_MODEL_NAME>-grafana``. Here, ``JUJU_MODEL_NAME``
+is the name of the Juju model on which you deployed your web app.
```

Comment:
I think we might want to consider spreading this out a bit more. Something like this:

```suggestion
From the list of URLs, look for the endpoint that contains a ``grafana`` suffix.
It has the format:

.. terminal::

    http://<IP_ADDRESS>/<JUJU_MODEL_NAME>-grafana

Here, ``<JUJU_MODEL_NAME>`` is the name of the Juju model on which you deployed
your web app.
```